frontend/src/concepts/modelCatalog/mockData/catalogModelMock.ts

```diff
@@ -0,0 +1,42 @@
+// This file is a workaround for webpack module parse error and will remove it as a part of RHOAIENG-18959
+import { ArtifactsProtocol, CatalogModel } from '~/concepts/modelCatalog/types';
+
+export const catalogModelMock = (partial?: Partial<CatalogModel>): CatalogModel => ({
+  repository: 'rhelai1',
+  name: 'granite-8b-code-base',
+  provider: 'IBM',
+  description: 'A decoder-only code model designed for code generative tasks',
+  longDescription: ` Granite-8B-Code-Base is a decoder-only code model designed for code
+    generative tasks (e.g., code generation, code explanation, code fixing,
+    etc.). It is trained from scratch with a two-phase training strategy. In
+    phase 1, our model is trained on 4 trillion tokens sourced from 116
+    programming languages, ensuring a comprehensive understanding of
+    programming languages and syntax. In phase 2, our model is trained on 500
+    billion tokens with a carefully designed mixture of high-quality data from
+    code and natural language domains to improve the models’ ability to reason
+    and follow instructions.`,
+  logo: 'data:image/gif;base64,R0lGODlhIAAgAMZLAAAAAAEAAAIAAAQAAAYAAAcAAAgAAAoAAAsAAA0AAA4AAA8AABEAABMAABwAAB4AAB8AACAAACQAACYAACkAAC8AADAAADsAAEIAAEMAAEkAAEoAAE4AAFAAAFYAAFkAAF8AAHAAAHMAAHQAAHsAAHwAAIcAAJMAAJcAAJoAAKYAAKwAAK0AAK4AALAAALYAALcAALkAALoAAL4AAMAAAMUAAMgAAMsAANAAANEAANcAANgAANoAANwAAOEAAOUAAOcAAOgAAOkAAOoAAOwAAO0AAO4AAO8AAPAAAPEAAPIAAP///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////yH5BAEKAH8ALAAAAAAgACAAAAf3gH+Cg4SFhoeIiYqLjI2Oj5CCRkaDk5SRhZaam5iSm5+TkaCjl46ko4+noKaqn441Sa2cixobPbGyoYeWRB0AHTm4sruWRyEAABYqQ8KkRYabSDYTyAkfMUJKSKNHQCaZ0SwRyAAMHCk4QklKSknrSkQpGj6EoEgyFwLkAw8aIygsYMRwccKDAgo9jtT7lIRHCQgByCELQAABAgMRAWD4caQUtyM3SFQwILEkAAcrtun6QwoJkh4sRGSQ0OBAAQMLJoCY0VFTJVJH3AXRUeNFCxc0dhRpttLTqaDvkvR0BS7XKURWUS3KaimV1U4sSYEdS7as2bKBAAA7',
+  readme: `# Granite-3.1-8B-Instruct
+    **Model Summary:**
+    Granite-3.1-8B-Instruct is a 8B parameter long-context instruct model finetuned from Granite-3.1-8B-Base using a combination of open source instruction datasets with permissive license and internally collected synthetic datasets tailored for solving long context problems. This model is developed using a diverse set of techniques with a structured chat format, including supervised finetuning, model alignment using reinforcement learning, and model merging.
```

Comment:
@ppadti would you mind updating the mocks with one of the updated samples from https://issues.redhat.com/browse/RHOAIENG-19096? @pboyd provided an updated sample today. We should also make sure we're following that updated file as the source of truth for our schema/types.